### PR TITLE
Dev ux

### PR DESCRIPTION
**Pull request by DavidMinarsch**
Massively simplified the configuration into one file.

Extracted scripts into tomte.

Still to do:

-> port to autonomy: @angrybayblade suggest you port this for 0.10.5 still.
check_doc_ipfs_hashes
check_packages

-> port to tomte: @jmoreira-valory 
spell-check

-> py.typed marker for all plugins needed @angrybayblade simply add them to all plugins in the next release cycle

-> We could remove package deps from Pipfile & tox.ini and have them auto-installed: a plugin could be created that simply loops over the deps in packages and dynamically sets those as it's own dependencies. Then the dev will not have to manually do this. They can always manually do it if they want to. @angrybayblade 

-> We can eventually create a default tox.ini which exists at tomte level and would simplify things even further

The aim is to have no `scripts` folder and at most one config file per repo. I would also like to get rid of the Makefile eventually.

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -1,18 +0,0 @@
-[MASTER]
```
Comment (DavidMinarsch) [inline .pylintrc:1]: part of tox.ini now

File: Makefile [hunk 2]
```diff
@@ -48,8 +48,7 @@ clean-test:
 # black: format files according to the pep standards
 .PHONY: formatters
```
Comment (DavidMinarsch) [inline Makefile:49]: Plan is to get fully rid of makefile

File: liccheck.ini [hunk 3]
```diff
@@ -1,106 +0,0 @@
-; some useful links:
```
Comment (DavidMinarsch) [inline liccheck.ini:1]: part of tox.ini now

File: pytest.ini [hunk 4]
```diff
@@ -1,13 +0,0 @@
-[pytest]
```
Comment (DavidMinarsch) [inline pytest.ini:1]: part of tox.ini now

File: scripts/check_copyright.py [hunk 5]
```diff
@@ -1,344 +0,0 @@
-#!/usr/bin/env python3
```
Comment (DavidMinarsch) [inline scripts/check_copyright.py:1]: part of tomte now

File: scripts/check_doc_links.py [hunk 6]
```diff
@@ -1,205 +0,0 @@
-#!/usr/bin/env python3
```
Comment (DavidMinarsch) [inline scripts/check_doc_links.py:1]: part of tomte now

File: scripts/check_readme.py [hunk 7]
```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env python3
```
Comment (DavidMinarsch) [inline scripts/check_readme.py:1]: part of tomte now

File: scripts/freeze_dependencies.py [hunk 8]
```diff
@@ -1,45 +0,0 @@
-#!/usr/bin/env python3
```
Comment (DavidMinarsch) [inline scripts/freeze_dependencies.py:1]: part of tomte now

File: setup.cfg [hunk 9]
```diff
@@ -1,101 +0,0 @@
-[flake8]
```
Comment (DavidMinarsch) [inline setup.cfg:1]: part of tox.ini now

File: sync.sh [hunk 10]
```diff
@@ -1,38 +0,0 @@
-rm -rf packages/open_aea
```
Comment (DavidMinarsch) [inline sync.sh:1]: no longer needed

File: upload.py [hunk 11]
```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env python3
```
Comment (DavidMinarsch) [inline upload.py:1]: no longer needed

**Review by dvilelaf — APPROVED** [review]

**Review by Adamantios — APPROVED** [review]

**Review by 0xArdi — APPROVED** [review]

**Pull request merged**